control/control.rnc

```diff
@@ -105,76 +105,79 @@ textdomain = element textdomain { STRING }
     | dhclient_set_hostname
-    | run_init_scripts_in_parallel
-    | inform_about_suboptimal_distribution
-    | skip_language_dialog
+    | disable_os_prober
+    | disable_register_w3m
+    | display_register_forcereg
+    | displaymanager_shutdown
+    | enable_autologin
+    | enable_autologin
     | enable_clone
-    | save_y2logs
+    | enable_firewall
+    | enable_kdump
+    | enable_local_users
     | enable_register_hwdata
     | enable_register_optional
-    | disable_os_prober
-    | display_register_forcereg
-    | disable_register_w3m
-    | register_monthly
+    | enable_sshd
+    | firewall_enable_ssh
+    | full_system_download_url
+    | full_system_media_name
+    | incomplete_translation_treshold
+    | inform_about_suboptimal_distribution
+    | installation_layout
+    | installation_ui
+    | kexec_reboot
+    | language
+    | lsm
     | manual_online_update
-    | root_password_as_first_user
-    | root_password_ca_check
-    | enable_local_users
-    | show_online_repositories
     | online_repositories_default
-    | show_addons
-    | addons_default
-    | kexec_reboot
-    | rle_offer_rulevel_4
-    | default_ntp_setup
-    | save_instsys_content
-    | displaymanager_shutdown
-    | enable_kdump
     | polkit_default_privs
-    | debug_workflow
-    | subworkflows
-    | default_subworkflow
-    | language
-    | timezone
+    | precise_time
+    | propose_hibernation
     | readonly_language
     | readonly_timezone
-    | enable_autologin
-    | default_target
+    | register_monthly
     | require_registration
-    | services_proposal
-    | debug_deploying
-    | show_drivers_info
-    | self_update_url
+    | rle_offer_rulevel_4
+    | root_password_as_first_user
+    | root_password_ca_check
+    | run_init_scripts_in_parallel
+    | save_instsys_content
+    | save_y2logs
     | self_update_id
+    | self_update_url
     | self_update_version
-    | installation_ui
-    | installation_layout
-    | cpu_mitigations
-    | full_system_media_name
-    | full_system_download_url
-    | precise_time
-    | propose_hibernation
-    | default_ntp_servers
-    | lsm
     # (deprecated) but make it backward compatible (bsc#1194403)
     | selinux
+    | services_proposal
+    | show_addons
+    | show_drivers_info
+    | show_online_repositories
+    | skip_language_dialog
+    | subworkflows
+    | timezone
+    | ui_mode
+    | write_hostname_to_hosts
 
 ## Default kernel parameters proposed by bootloader
 additional_kernel_parameters =		element additional_kernel_parameters { STRING }
 ## How much per cent is the minimum coverage of translations to consider
 ## it being 'complete'. Less are reported if user tries to switch to such
 ## language during installation / upgrade.
 incomplete_translation_treshold =	element incomplete_translation_treshold { STRING }
+## Default proposed timeout settings for bootloader (jsc#SLE-22667)
+boot_timeout =                          element boot_timeout { INTEGER }
```

Comment:
NP: why so indented?

Reply:
well, my intention is to have same indentation as rest, but looks like again there is some tabs :(

Reply:
> well, my intention is to have same indentation as rest, but looks like again there is some tabs :(

Yep, looks like. I asked because the lines above and below does not have the same indentation.